src/games/gameitem.py

```diff
@@ -286,30 +363,31 @@ def _observers_tooltip(self, observers):
         if not observers:
             return ""
 
-        observer_fmt = "{country_icon} {observer}"
+        obs_table_start = "<table><tr><td valign='middle'>Observers:</td>"
+        obs_table_end = "</td></tr></table>"
+        observer_fmt = "<td>{country_icon}</td><td valign='middle'>{observer}"
 
         observer_strings = [observer_fmt.format(
             country_icon=self._country_icon_fmt(observer),
             observer=observer.login)
             for observer in observers]
-        return "Observers: " + ", ".join(observer_strings)
+        return obs_table_start + ",</td>".join(observer_strings) + obs_table_end
 
     def _mods_tooltip(self, mods):
         if not mods:
             return ""
-        return "<br/>With: " + "<br/>".join(mods.values())
+        mods_start = "<br/>With Sim Mods:"
+        mods_inst = "<br/><font color='green'><i>Installed:</i><br/>" + \
+                    "<br/>".join(mods[0]) + "</font>" if mods[0] else ""
```

Comment:
Instead of using `mods[0]` for installed mods and `mods[1]` for not-installed mods, it would be much less insane to use a named dictionary field. So people don't have to stare at this for 30 seconds before they figure out what the hell is going on, as I just did.

You could go one better and make it an actual object, and enforce that it's never `None`. This would allow you to avoid the checks (since `join()` with empty-list yields empty-string).